mandatory.c

```diff
@@ -30,3 +30,32 @@ int _putchar(int c)
 {
 	return (write(1, &c, 1));
 }
+
+/**
+ * print_num - prints an integer in (base 10) to the stdout and return
+ * the number of characters printed.
+ * @n: the number to be printed.
+ * @base: the base number of decimal numbers.
+ *
+ * Return: On success, the number of bytes printed is returned.
+ */
+int print_num(int n, int base)
+{
+	int counter = 0;
+	char *numbers = "0123456789";
+
+	if (n < 0)
+	{
+		write(1, "-", 1);
+		return (print_num(-n, base) + 1);
+	}
+	else if (n < base)
+		return (_putchar(numbers[n]));
+	else if (n > base)
+	{
+		counter = print_num(n / base, base);
+		return (counter + (print_num(n % base, base)));
+	}
+	else
+		return (-1);
```

Comment:
There is no need for an else just `return (-1)` is enough.
the same as above, an if statement with a return may not necessarily need an else following it.

Reply:
i didn't quite understand your point about the above else if statements